### PR TITLE
Second

### PR DESCRIPTION
**Pull request by trishas95**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
  
  

---

### Related issue

- #146

### CODE REVIEW

File: frontend/i18n/en-US.json [hunk 1]
```diff
@@ -37,5 +37,9 @@
   "about": "About",
   "supporters": "Supporters",
   "impress": "Impress",
-  "theme": "Theme"
+  "theme": "Theme",
+  "wikimedia-de-logo-label": "Link to the website of Wikimedia Deutschland.",
+  "wikimedia-rs-logo-label": "Link to the website of Wikimedia Serbia",
```
Comment (andrewtavis) [inline frontend/i18n/en-US.json:42]: Could you add a period here for consistency?

File: frontend/i18n/en-US.json [hunk 2]
```diff
@@ -37,5 +37,9 @@
...
+  "impacthub-belgrade-logo-label": "Link to the website of Impact Hub Belgrade"
```
Comment (andrewtavis) [inline frontend/i18n/en-US.json:43]: A period here too would be great :)

**Review by andrewtavis — CHANGES_REQUESTED** [review]
Thanks @trishas95! This is basically done. The last thing that's needed is that you need to add a `:` before the aria labels so that the translated values can be read in rather than a string. I'm realizing I should have explained this better. I just pulled in your branch and the correct way of doing this would be:

```
:aria-label="$t('wikimedia-de-logo-label')"
```

Once the fixes I mentioned about adding some periods in the descriptions and changing the `aria-label` attributes is done we'll be good to merge!

**Comment by trishas95** [comment]
Hello @andrewtavis, I made the changes you've mentioned above. This is one my first few PRs, looking forward to learning more! :)

**Comment by andrewtavis** [comment]
Thank you, @trishas95! Happy to help you learn more :) I’ll do the final review tomorrow 😊

**Comment by andrewtavis** [comment]
Hey @trishas95! Have had a few busy days and didn't get a moment to do the final check of this till now. I'm realizing there's still one very small issue here 🙃 You need the elements within the `t()` to be in single quotes, so `t('wikimedia-de-logo-label')`. Sorry I didn't mention this part more explicitly when I discussed the `:` above. As of now what we have there is `translate(wikimedia - de - logo - label)` like it's subtraction, but what we want is `translate('the-string-in-kebab-case')` that will then feed the key into translation and provide the appropriate value :)

**Comment by trishas95** [comment]
@andrewtavis I made the changes :D

**Pull request merged**